.github/workflows/diff_prs.yml

```diff
@@ -0,0 +1,53 @@
+name: Diff as PRs
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch to compare from (default: develop).'
+        default: 'develop'
+        required: false
+      target_branch:
+        description: 'Target branch to compare to.'
+        required: true
+
+permissions:
+  contents: read
+
+jobs:
+  fetch-prs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Fetch Merged PRs Between Branches
+        uses: ./.github/actions/pr_diff
+        with:
+          base_branch: ${{ github.event.inputs.base_branch }}
+          target_branch: ${{ github.event.inputs.target_branch }}
+
+      - name: Format output
```

Comment:
Why not do this part in the action itself ? or as a pr_diff_logger action

Reply:
I wanted to keep the other action very generic (outputs json) and make the formatting in another step.

Putting all of this in a dedicated action seemed a bit overkill though, especially for a workflow that little.
Note: this is using the `$GITHUB_STEP_SUMMARY` variable which generates (supposedly :crossed_fingers: ) a standalone report for the workflow execution

Reply:
Ok, the 2 sentences seem a bit contradictory (generic action but not generic output) but it's fine 👍